src/modules/Security.rb

```diff
@@ -363,6 +340,80 @@ def read_kernel_settings
         val = default_value if val.nil? || val == ""
         @Settings[key] = val
       end
+
+      log.debug "Settings (after #{__callee__}): #{@Settings}"
```

Comment:
Interesting trick... :open_mouth: 

The method names are pretty descriptive in this case, otherwise I'd rather prefer some human readable string than the internal function names.